official/recommendation/README.md

```diff
@@ -0,0 +1,35 @@
+# Recommendation Model
+This is an implementation of the Neural Collaborative Filtering (NCF) framework with Neural Matrix Factorization (NeuMF) model as described in the [Neural Collaborative Filtering](https://arxiv.org/abs/1708.05031) paper. Current implementation is based on the code from the authors' [NCF code](https://github.com/hexiangnan/neural_collaborative_filtering) and the Standford pytorch implementation at [mlperf repo](https://github.com/mlperf/reference/tree/master/recommendation/pytorch).
```

Comment:
Probably should change "pytorch" to "PyTorch" which is their official name.

Reply:
Will do. Thank you for the carful checking.